sentry_sdk/client.py

```diff
@@ -74,23 +74,37 @@ class _Client(object):
     """
 
     def __init__(self, *args, **kwargs):
-        # type: (*Optional[str], **Any) -> None
+        # type: (*Any, **Any) -> None
```

Comment:
Not sure what happened here but this diff seems to be the result of merge conflict resolution and not actually intended.